### PR TITLE
Mantl has a dedicated rpm repository per release

### PR DESCRIPTION
**Pull request by Zogg**
Comments and suggestions are welcome.

WARNING: do not merge until the package repositories are populated with packages

### CODE REVIEW

**Comment by siddharthist** [comment]
We should restart the CI build now that the repo is populated

File: roles/repos/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+mantl_version: 1.1.0
```
Comment (ryane) [inline roles/repos/defaults/main.yml]: should this be 1.2 to correspond with https://bintray.com/shippedrepos/mantl-1.2?

Comment (Zogg) [inline roles/repos/defaults/main.yml]: Yes. I'm using a GUI to push git changes, and apparently there were conflicts and my changes were not pushed to the remote branch... I'll sort it out, thanks for a notice.

**Comment by Zogg** [comment]
Something screwed up the VCS. Opening a new PR with a clean branch.

**Pull request closed without merging**